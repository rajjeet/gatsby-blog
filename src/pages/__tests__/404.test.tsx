import React from 'react';
import { render, cleanup } from '@testing-library/react';
import NotFoundPage from '../404';

afterEach(cleanup);

describe('404 Page', () => {
  it('should render', () => {
    const { asFragment } = render(<NotFoundPage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
