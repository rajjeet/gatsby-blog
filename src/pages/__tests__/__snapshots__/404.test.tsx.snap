// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 Page should render 1`] = `
<DocumentFragment>
  <div
    class="sc-htpNat faJHDz"
  >
    <div
      class="sc-bwzfXH bmdiOC"
    >
      <a
        aria-label="See Home"
        href="/"
      >
        <div
          class=" gatsby-image-wrapper"
          style="position: relative; overflow: hidden; width: 3em;"
        >
          <div
            style="width: 100%; padding-bottom: 100%;"
          />
          <picture>
            <source
              sizes=""
              srcset=""
            />
            <img
              alt="Ortmesh logo"
              loading="lazy"
              sizes=""
              src=""
              srcset=""
              style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; object-fit: cover; object-position: center; opacity: 0; transition: opacity 500ms;"
            />
          </picture>
          <noscript>
            &lt;picture&gt;&lt;source srcset="" /&gt;&lt;img loading="lazy" src="" alt="Ortmesh logo" style="position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center"/&gt;&lt;/picture&gt;
          </noscript>
        </div>
      </a>
      <a
        href="/"
      >
        <h2>
          Ortmesh
        </h2>
        <p>
          Write code that matters
        </p>
      </a>
      <button
        class="sc-bdVaJa jCUGPA"
      >
        About
      </button>
    </div>
    <h4>
      Error 404:
    </h4>
    <p>
      This page is not found. Please enter a valid URL path into the address bar.
    </p>
    <a
      href="/"
    >
      Go to Home
    </a>
  </div>
</DocumentFragment>
`;
